eessi_bot_software_layer.py

```diff
@@ -78,15 +89,15 @@ def main():
 
     if opts.file:
         event = read_event_from_json(opts.file)
-        log_event(event)
-        handle_event(event)
+        event_info = get_event_info(event)
+        handle_event(event_info)
     elif opts.cron:
         log("Running in cron mode")
     else:
         # Run as web app
-        app = create_app()
-        app.run()
-
+        app = create_app(klass=EESSIBotSoftwareLayer)
+        log("EESSI bot for software layer started!")
+        waitress.serve(app, listen='*:3000')
 
 if __name__ == '__main__':
     main()
```

Comment:
Starting the bot currently doesn't produce any output at all, ideally it should print something to signal that the bot has been started successfully.
A useful message would be to print the location to the log, for example (which should probably be taken care of by the `start` method of the instance that is produced by PyGHee's `create_app` (so it can be fixed as a part of #9)